Monika After Story/game/script-topics.rpy

```diff
@@ -3781,14 +3781,17 @@ init 5 python:
     addEvent(Event(persistent.event_database,eventlabel="monika_ribbon",category=['monika'],prompt="Ribbons",random=True))
 
 label monika_ribbon:
-    if monika_chr.hair.name != "def":
+    if monika_chr.get_acs_of_type('ribbon') is None:
```

Comment:
```suggestion
    if monika_chr.is_wearing_acs_type('ribbon'):
```
this exists and we should use it if we dont need the object

Reply:
should be if not, not just if